tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -42,4 +49,49 @@ Dashboard Test Setup
   Wait for RHODS Dashboard to Load
 
 Dashboard Test Teardown
-  Close All Browsers
\ No newline at end of file
+  Close All Browsers
+  
+Get HTTP Status Code
+    [Arguments]  ${link_to_check}
+    ${response}=    RequestsLibrary.GET  ${link_to_check}   expected_status=any
+    Run Keyword And Continue On Failure  Status Should Be  200
+    [Return]  ${response.status_code}
+
+Check All Links Are working
+  [Documentation]  Verify the all links after appling a filter
+  ${link_elements}=  Get WebElements    //a[@class="odh-card__footer__link" and not(starts-with(@href, '#'))]
```

Comment:
you don't need to check these links because there are already verified by `Verify Resource Link Http status code` test case in the same suite